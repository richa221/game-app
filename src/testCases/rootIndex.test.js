import Game from '../containers/game'
import { Provider } from "react-redux";
import configureStore from "redux-mock-store";
import Adapter from 'enzyme-adapter-react-16'
import { shallow,configure } from 'enzyme'
configure({ adapter: new Adapter() })

describe('Game snapshot tests', () => {
    const mockStore = configureStore();
   
    it('renders correctly', () => {  
     const tree = shallow(<Provider store={mockStore({})}><Game /></Provider>)
     expect(tree).toMatchSnapshot();
    })
   });