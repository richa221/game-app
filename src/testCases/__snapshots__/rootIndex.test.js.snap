// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game snapshot tests renders correctly 1`] = `ShallowWrapper {}`;
